src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolderDescriptor.java

```diff
@@ -41,6 +41,40 @@ public String getDisplayName() {
     }
 
     /**
+     * Needed if it wants Folders are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     * TODO: Replace to {@code NestedProjectsCategory.ID}
+     *
+     * @return A string it represents a ItemCategory identifier.
+     */
+    public String getCategoryId() {
+        return "nestedprojects";
+    }
+
+    /**
+     * Needed if it wants Folders are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string with the Item description.
+     */
+    public String getDescription() {
+        return Messages.Folder_Description();
+    }
+
+    /**
+     * Needed if it wants Folder are categorized in Jenkins 2.x.
+     *
+     * TODO: Override when the baseline is upgraded to 2.x
+     *
+     * @return A string it represents a URL pattern to get the Item icon in different sizes.
+     */
+    public String getIconFilePathPattern() {
+        return "plugin/cloudbees-folder/images/:size/folder.png";
```

Comment:
Why are you not just using the `images/:size/folder.png` in Jenkins core? ![`folder.png`](https://raw.githubusercontent.com/jenkinsci/jenkins/be9bc0dba552e271b47102292fc9f585afaaf212/war/src/main/webapp/images/48x48/folder.png)